ddtrace/contrib/celery/task.py

```diff
@@ -84,6 +84,12 @@ def _task_init(func, task, args, kwargs):
 
 @require_pin
 def _task_run(pin, func, task, args, kwargs):
+    if task.request.hostname is not None:
+        try:
+            global WORKER_SERVICE
+            WORKER_SERVICE = 'celery_{}'.format(task.request.hostname.split('@')[0])
+        except AttributeError:
+            pass
```

Comment:
This seems unrelated to the topic of the PR.

I also don't think you want to change the global value of `WORKER_SERVICE` here or you'll end up updating it on a per-app basis rather than per-task.

What's the name of `hostname` usually?